.github/workflows/lint.yml

```
@@ -41,3 +41,14 @@ jobs:
         with:
           command:      fmt
           args:         --all -- --check
+  check-spellcheck:
+    name:               Check For Spelling and/or Grammar Mistakes
+    runs-on:            ubuntu-latest
+    env:
+      RUST_BACKTRACE:   full
+    steps:
+      - name:           Add cargo-spellcheck
+        run:            rustup component add cargo-spellcheck
+
+      - name:           Run cargo-spellcheck
+        run:            cargo spellcheck check
```

```suggestion
    steps:
      - name:           Cancel Previous Runs
        uses:           styfle/cancel-workflow-action@0.4.1
        with:
          access_token: ${{ github.token }}
      
      - name:           Checkout sources & submodules
        uses:           actions/checkout@master
        with:
          fetch-depth:  5
          submodules:   recursive
      
      - name:           Add rustfmt
        run:            rustup component add cargo-spellcheck
      
      - name:           rust-fmt check
        uses:           actions-rs/cargo@master
        with:
          command:      spellcheck
          args:         check
```

Should be something like that.